mlflow/server/js/src/shared/web-shared/genai-traces-table/cellRenderers/SessionIdLinkWrapper.tsx

```diff
@@ -1,25 +1,29 @@
-import { shouldEnableChatSessionsTab } from '../utils/FeatureUtils';
 import MlflowUtils from '../utils/MlflowUtils';
 import { Link } from '../utils/RoutingUtils';
 
 export const SessionIdLinkWrapper = ({
   sessionId,
   experimentId,
+  traceId,
   children,
 }: {
   sessionId: string;
   experimentId: string;
+  traceId?: string;
   children: React.ReactElement;
 }) => {
-  if (shouldEnableChatSessionsTab()) {
-    return (
-      <Link
-        // prettier-ignore
-        to={MlflowUtils.getExperimentChatSessionPageRoute(experimentId, sessionId)}
-      >
-        {children}
-      </Link>
-    );
-  }
-  return children;
+  const url = traceId
+    ? `${MlflowUtils.getExperimentChatSessionPageRoute(experimentId, sessionId)}?selectedTraceId=${encodeURIComponent(
+        traceId,
+      )}`
+    : MlflowUtils.getExperimentChatSessionPageRoute(experimentId, sessionId);
```

Comment:
related to above can we use `URLSearchParams` and get rid of the ternary if traceId doesn't have to be nullable? also nit: maybe define and export a const for `selectedTraceId` so it can be imported and used in the sessions tab as well.

```suggestion
  const searchParams = new URLSearchParams({ selectedTraceId: traceId } );
  const url = `${MlflowUtils.getExperimentChatSessionPageRoute(experimentId, sessionId)}?${searchParams.toString()}`
```

Reply:
Totally - done!